### PR TITLE
README for DMTF dir

### PR DESCRIPTION
**Pull request by bcornec**
This patch adds the following:
- a README file for the dmtf dir on how to use the mockup
- a Dockerfile in order to launch example for python-redfish in a container
- a small patch for comment usage

### CODE REVIEW

**Comment by bcornec** [comment]
More documentation was needed, so I made this README to help starting with the usage of the DMTF simulator. Some additional small fixes.

File: examples/docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:15.04
+MAINTAINER bruno.cornec@hp.com
+ENV DEBIAN_FRONTEND noninterative
+ENV http_proxy http://web-proxy.fra.hp.com:8080
+ENV https_proxy http://web-proxy.fra.hp.com:8080
+# Install deps for Redfish mockup
+RUN apt-get update
+RUN apt-get -y install python-mock python-pip git openssh-client libpython2.7-dev python-oslotest
+RUN apt-get -y ansible
+RUN useradd -m bruno
```
Comment (AevaOnline) [inline examples/docker/Dockerfile:10]: even though you're the MAINTAINER, I don't think your user name should get added to everyone's docker container ...

File: examples/docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:15.04
+MAINTAINER bruno.cornec@hp.com
+ENV DEBIAN_FRONTEND noninterative
+ENV http_proxy http://web-proxy.fra.hp.com:8080
```
Comment (AevaOnline) [inline examples/docker/Dockerfile:4]: HP's internal proxy services don't belong in the public repo

Comment (bcornec) [inline examples/docker/Dockerfile:4]: Right. Could replace by a commented example

File: examples/docker/Dockerfile (hunk 1 above)
Comment (bcornec) [inline examples/docker/Dockerfile:10]: Right again.

**Pull request closed without merging**